sdk/tests/pow/miner.rs

```diff
@@ -11,23 +11,48 @@ use iota_sdk::{
 
 #[test]
 fn get_miner_score() {
-    let miner = get_miner(4000);
+    let miner = get_miner(100);
     let mut bytes = rand_bytes(256);
 
     let nonce = miner(&bytes[0..248]).unwrap();
     bytes[248..].copy_from_slice(&nonce.to_le_bytes());
 
-    assert!(PowScorer::new().score(&bytes) >= 4000f64);
+    assert!(PowScorer::new().score(&bytes) >= 10f64);
```

Comment:
You changed the score here but not in the builder?

Reply:
Yes, thanks

Reply:
Is 10 not so small it can be achieved by accident?

Reply:
Everything can be achieved by accident with some probability :trollface: 
But yeah 100 should be fine as well, undid the change